### ISSUE TITLE
Unable to work with @sentry/nextjs

### ISSUE THREAD

**Issue by darrist** [getsentry/sentry-docs#3703]
Sentry recently have release @sentry/nextjs which helps targeting both server-side render and client-side render. However, the documentation in sentry.io/nextjs and this repo have yet been updated to provide a guideline yet.

I haven't find out how to use it either, I'm just writing so that the community is aware of this.

### Environment

How do you use Sentry?
Sentry on-premise

Which SDK and version?
```
    "next": "latest",
    "@sentry/nextjs": "^6.4.1",
    "@sentry/rrweb": "^0.3.1",
```

### Steps to Reproduce
1. Setup next.js repo using https://docs.sentry.io/platforms/javascript/guides/nextjs/
2. Setup rrweb using https://docs.sentry.io/platforms/javascript/guides/nextjs/configuration/integrations/rrweb/

### Actual Result

I'm getting problem such as `window is undefined`.

**Comment by dcramer** [comment]
Eek - thats not a great workflow. Will have someone track/re-open.

**Comment by PeloWriter** [comment]
@chadwhitacre - can you review our workflow; we don't want this type of good feedback to be closed.

**Comment by getsentry-release** [comment]
Routing to @getsentry/team-webplatform for [triage](https://develop.sentry.dev/processing-tickets/#3-triage). ⏲️

**Comment by lobsterkatie** [comment]
@darrist - Sorry you were having trouble! 

I'm wondering if you could be a little bit more specific with what you were looking for and didn't find. You link in your description to our docs, which do in fact have set-up and usage information, so forgive me, but I'm a little confused what you mean when you say "the docs haven't been updated to provide a guideline."

Thanks!

**Comment by darrist** [comment]
Hi peeps. Thanks for trying to help and pardon me for the vague title writing. I already found out that `@sentry/nextjs` is a drop-in replacement for `@nextjs/browser`

What I've faced is actually the [rrweb integration document](https://docs.sentry.io/platforms/javascript/guides/nextjs/configuration/integrations/rrweb/) is suggesting use of  `@sentry/browser` which will not simply work due to missing window API, and it is not indicating to use with sentry.server.js or sentry.client.js

So I think some correction on this document can remove some confusion.

However, I am still facing some 403 issues where its impossible to upload rrweb payload to on-premise Sentry.
